src/frontEnd/forms/InputForField.tsx

```diff
@@ -3,11 +3,15 @@ import { textChangeHandler } from "./textChangeHandler";
 import { TextField, Theme } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
+const textFieldStyle = window.innerWidth <= 480
```

Comment:
seems like this is a second place with some condition, maybe we should have an independent location where we say `isMobile = () => window.innerWidth <= 480` and then use that in several places

Reply:
u can also destructure this right away, `const {space, width} = isMobile() ? { width: "100%, space: 0 } : { width: "300px", space: 1 }` so that u needn't repeat `textFieldStyle` thrice below